workshops/templates/workshops/workshop_issues.html

```diff
@@ -36,7 +36,7 @@
         {% if event.mailto_ %}
           <a href="mailto:{{event.mailto_}}?subject={% filter urlencode %}Missing location for workshop {{event.get_ident}}{% endfilter %}&body={% filter urlencode %}Hi,
 
-Can you please send us the location where the {{ event.get_ident }} workshop took place?
+Can you please send us the location where the {{ event.get_ident }} workshop took place? We need a country, address, venue, and latitude/longitude.
```

Comment:
On Fri, Nov 20, 2015 at 08:56:17AM -0800, maneesha sane wrote:
> I'd want to understand why we are collecting this for past
> workshops.  Longitudinally I think we want to track workshops
> geographically i.e. how many in Florida? how many in south east US?
> etc.  But the question is how specific do we need to get? @jduckles
> ?

I think the motivation is:

1. Publishing more past workshops makes it easier to show folks how
   busy we are, or where we have been active in the past.
2. This PR requires the additional fields before an event is published
   [1,2].
3. We have to collect the newly-required information to avoid
   unpublishing old events that we used to publish.

If that reasoning makes sense, that means we need to be specific
enough to pass the new, stricter 'published' filter.

[1]: https://github.com/swcarpentry/amy/pull/567#issuecomment-156452337
[2]: https://github.com/swcarpentry/amy/pull/567/files#r45492564